foundation-extensions/adobe-experience-platform-assurance/README.md

```diff
@@ -406,11 +406,9 @@ class CoreStartCompletionCallback : Java.Lang.Object, IAdobeCallback
 The `startSession` API needs to be called to begin a Project Griffon session. When called, SDK displays a PIN authentication overlay to begin a session.
 
 {% hint style="info" %}
-Android does not require this API to be called. The AEP Assurance Android extension registers the app lifecycle handlers which automatically pick up any deep links and use them to start the session.
-{% endhint %}
+The AEP Assurance Android extension does not require this API to be called since it registers the app lifecycle handlers which automatically pick up any deep links and use them to start the session.
 
-{% hint style="info" %}
-You may call this API when the app launches with a url \(see code snippet below for sample usage\)
+For AEP Assurance iOS extension, you should call this API when the app launches with a url \(see code snippet below for sample usage\).
```

Comment:
Rewording and other small fixes:
- When using the AEP Assurance iOS extension, ...
- As Caleb would say, let's remove forward slashed \ before parentheses. :)